### PR TITLE
Create management application fixtures for local development (#239)

### PR DESCRIPTION
**Pull request by ssciolla**
Leveraging an existing script, this PR makes changes to the `management` application and introduces a new management command, `import_manage_fixtures`, so that the `management_*` tables in the MySQL database have data to use for testing. A task list describing the changes is provided below. This PR aims to resolve issue #239. 

- [x] Move existing code out of `scripts` folder, modify as necessary, and re-name to `import_fixtures.py`.
- [x] Create `management/fixtures` sub-directory and add two JSON files containing fixtures for students, mentors, and cohorts (all fabricated data).
- [x] Write convenient management command in `student_explorer` application that calls the `main` function of `import_fixtures.py`.
- [x] Use new management command in `start-localhost.sh`.

### CODE REVIEW

**Comment by jonespm** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           

Complexity increasing per file
==============================
- management/import_fixtures.py  3
- student_explorer/management/commands/import_manage_fixtures.py  2
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/ITS_Teaching_And_Learning/student_explorer/pullRequest?prid=5371063&bid=17604922)

**Comment by mfldavidson** [comment]
@ssciolla Codacy is mad you created the variable `created` but didn't use it

**Comment by ssciolla** [comment]
> @ssciolla Codacy is mad you created the variable `created` but didn't use it

Okay, I'll fix it. That was code from before that I essentially copied and pasted, but may as deal with it now. Part of the problem is the use of `get_or_create`, but I'll find some way to resolve it.

**Comment by mfldavidson** [comment]
I am not judging you. But maybe I will git blame and judge whoever wrote that code.

Jkjkjkjkjkjkjkjkjk

**Comment by jonespm** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- management/import_fixtures.py  3
- student_explorer/management/commands/import_manage_fixtures.py  2
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/ITS_Teaching_And_Learning/student_explorer/pullRequest?prid=5371063&bid=17604922)

File: management/import_fixtures.py [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+        student = Student.objects.get_or_create(username=student_id)[0]
+        mentor = Mentor.objects.get_or_create(username=mentor_id)[0]
+        StudentCohortMentor.objects.get_or_create(
+            student=student,
+            cohort=cohort,
+            mentor=mentor
+        )
+
+if __name__ == '__main__':
+    main()
```
Comment (mfldavidson) [inline management/import_fixtures.py:48]:
```suggestion
    main()
    
```
insert a newline at end?

Comment (ssciolla) [reply management/import_fixtures.py]: Sure

**Comment by jonespm** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           

Complexity increasing per file
==============================
- management/import_fixtures.py  3
- student_explorer/management/commands/import_manage_fixtures.py  2
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/ITS_Teaching_And_Learning/student_explorer/pullRequest?prid=5371063&bid=17604922)

**Comment by mfldavidson** [comment]
Ok wtf Codacy you wanted us to add the whitespace now you don't????????

**Comment by jonespm** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- management/import_fixtures.py  3
- student_explorer/management/commands/import_manage_fixtures.py  2
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/ITS_Teaching_And_Learning/student_explorer/pullRequest?prid=5371063&bid=17604922)

**Comment by ssciolla** [comment]
Yeah, I don't know what's going on. I thought there was an extra newline after committing your suggestion (I see a newline when I look at things in VSCode), but now it looks like there isn't? It's alright, Codacy seems happy for the moment.

**Comment by jonespm** [comment]
Hmm, did this old file "Student Explorer Data Master.xlsx" just not exist? I didn't even know about this process, but basically looks like what we wanted to do. 

Can't we just save it as 2 CSV's? It makes it easier to see the diffs over a binary file and it looks simple enough. I'm okay if you don't want to make this change though as it was pre-existing code. 

I ran the xlsx though https://codebeautify.org/excel-to-json and we it seems like we could pretty easily just use JSON too.

**Comment by ssciolla** [comment]
That old file wasn't part of the repository. I'm happy to change the input format, just let me know what you'd like.

**Comment by jonespm** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           

Complexity increasing per file
==============================
- management/import_fixtures.py  5
- student_explorer/management/commands/import_manage_fixtures.py  2
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/ITS_Teaching_And_Learning/student_explorer/pullRequest?prid=5371063&bid=17604922)

**Comment by jonespm** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- student_explorer/management/commands/import_manage_fixtures.py  2
- management/import_fixtures.py  5
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/ITS_Teaching_And_Learning/student_explorer/pullRequest?prid=5371063&bid=17604922)

**Comment by ssciolla** [comment]
Okay, @jonespm approved, so I'll merge so he can continue with his work. 

Ultimately, we might want to revisit how this `management/import_fixtures.py` file works, since I think it's flooding the logs on startup with more warnings, I think because it uses the `get_wsgi_application` from Django. There may be another, better way to import the rest of the applications. The logging could also be improved; I believe I had some issues with using a standard logging initialization.

But it's working for now.

**Pull request merged**